src/code/z_play.c

```diff
@@ -21,7 +22,6 @@ u8 sMotionBlurStatus;
 #include "buffers.h"
 #include "idle.h"
 #include "sys_cfb.h"
-#include "z64bombers_notebook.h"
```

Comment:
why is this removed from z_play?

Reply:
Not sure, probably just an accident. Will add it back.